dataland-frontend/tests/e2e/utils/EuTaxonomyFinancialsUpload.ts

```diff
@@ -27,27 +33,101 @@ export function submitEuTaxonomyFinancialsUploadForm(): Cypress.Chainable {
  * @param data the data to fill the form with
  */
 export function fillEuTaxonomyForFinancialsUploadForm(data: EuTaxonomyDataForFinancials): void {
```

Comment:
I suggest renaming this function.

Reason:  Before this PR this function was solely used to fill the form, but now it actually tests and validates stuff.
It is not a plain "uploader" function anymore.

So maybe `fillAndValidateEuTaxonomyForFInancialsUploadForm()`

Reply:
Note: Also check out my other comment in this file, which suggest outsourcing some of the test code to a component test.

Reply:
ok

Reply:
On it

Reply:
Method was renamed. As discussed no further refactoring.